### DISCUSSION TITLE
Dateizugrif erlauben Android Tablet

### DISCUSSION THREAD

**Discussion by tobsi** [community/community#124056] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hallo zusammen.

Das Codieren ist für mich eine neue Herausforderung, die ich erlernen möchte. Der Start, bevor ich überhaupt ein Programm schreiben kann, bereitet mir jedoch Schwierigkeiten.
Auf meinem Android Tablet habe ich VSCode heruntergeladen. Das funktioniert alles, nur habe ich bisher noch nicht herausgefunden, wie ich erlauben kann, dass VSCode auf meine Dateien, besser gesagt meine Ordner zugreifen kann. Weis da jemand, wie das möglich ist, oder hatte schon ähnliche Probleme, die mir wenigstens ein bisschen helfen können?

**Comment by ebndev** [comment]
Hi @tobsi, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
